WMF Framework/CommonStrings.swift

```diff
@@ -146,4 +146,8 @@ public class CommonStrings: NSObject {
     @objc public static let appStoreShortDescription = WMFLocalizedString("app-store-short-description", value: "Download the Wikipedia app to explore places near you, sync articles to read offline and customize your reading experience.", comment: "Short description of the app for the app store")
     @objc public static let appStoreReleaseNotes = WMFLocalizedString("app-store-release-notes", value: "Fully customizable and easier to read Explore feed. Localization, performance improvements and bug fixes.", comment: "Short summary of what is new in this version of the app for the app store")
     @objc public static let appStoreKeywords = WMFLocalizedString("app-store-keywords", value: "Wikipedia, reference, wiki, encyclopedia, info, knowledge, research, information, explore, learn", comment: "Short list of keywords describing the app for the app store. It is required that these are individual words, not phrases, and are comma separated.")
+    
+    @objc public static let editAttribution = WMFLocalizedStringWithDefaultValue("wikitext-upload-save-anonymously-warning", nil, nil, "Edits will be attributed to the IP address of your device. If you %1$@ you will have more privacy.", "Button sub-text informing user or draw-backs of not signing in before saving wikitext. Parameters:\n* %1$@ - sign in button text")
```

Comment:
Is there precedent for using the parameter to fill in other localized text or some guidance from Apple that says this OK? There are likely cases where the item localized separately is different than how it would fit in the full sentence.

Reply:
good question - iirc we did this in the section editing workflow, but i'll double-check and ponder...

Reply:
@joewalsh 
![screen shot 2018-09-25 at 10 29 17 am](https://user-images.githubusercontent.com/3143487/46031550-fd849580-c0ad-11e8-994c-6d478223fb3b.png)

So to achieve ^ w/o composition... hmm... (editing use composition too)... so maybe a single string with html links then a web view and intercept those link taps?